aptoide-views/build.gradle

```diff
@@ -0,0 +1,123 @@
+apply plugin: 'com.android.library'
+apply plugin: 'kotlin-android'
+apply plugin: 'kotlin-android-extensions'
+android {
+
+  compileSdkVersion Integer.parseInt(project.COMPILE_SDK_VERSION)
+  buildToolsVersion BUILD_TOOLS_VERSION
+
+  defaultConfig {
+    minSdkVersion project.LIB_MINIMUM_SDK_VERSION
+    targetSdkVersion Integer.parseInt(project.TARGET_SDK_VERSION)
+    multiDexEnabled true
+    testInstrumentationRunner "android.support.test.runner.AndroidJUnitRunner"
+    vectorDrawables.useSupportLibrary = true
+  }
+
+  compileOptions {
+    sourceCompatibility JavaVersion.VERSION_1_8
+    targetCompatibility JavaVersion.VERSION_1_8
+  }
+
+  flavorDimensions "product", "mode"
+  productFlavors {
+    cobrand { dimension "product" }
+    vanilla { dimension "product" }
+    internal { dimension "mode" }
+    dev { dimension "mode" }
+    prod { dimension "mode" }
+    leak { dimension "mode" }
+    monetisation { dimension "mode" }
+  }
+
+  buildTypes {
+    debug {
+      resValue "string", "authenticator_account_type", "compat"
```

Comment:
I'll remove most of these as you suggested, as I agree we can maybe find a better way later, but most of these exist due to missing resources when trying to run tests in the module 😞. For example, this one was because the compiler complained about missing the "authenticator_account_type" string that is injected here.

I'm really not sure if we can fix this all of this elegantly but we can try later 👍